core/ui/ViewTemplate/subtitle.tid

```diff
@@ -4,7 +4,11 @@ tags: $:/tags/ViewTemplate
 \whitespace trim
 <$reveal type="nomatch" stateTitle=<<folded-state>> text="hide" tag="div" retain="yes" animate="yes">
 <div class="tc-subtitle">
-<$link to={{!!modifier}} />
-<$view field="modified" format="date" template={{$:/language/Tiddler/DateFormat}}/>
+<$list filter="[all[shadows+tiddlers]tag[$:/tags/ViewTemplate/Subtitle]!has[draft.of]]" variable="subtitleTiddler" counter="indexSubtitleTiddler">
```

Comment:
Could we start using the form `!is[draft]` instead of `!has[draft.of]` when introducing new changes to the core templates?